feat(tem): scale plan MTA-5622 (#4576)
scaleway/docs-content: pages/transactional-email/reference-content/tem-dedicated-ip.mdx

---
meta:
  title: Understanding managed dedicated IPs
  description: Learn about the benefits of using a dedicated IP for email sending, how TEM-managed dedicated IPs work, and when to use them.
content:
  h1: Understanding managed dedicated IPs
  paragraph: Learn about the benefits of using a dedicated IP for email sending, how TEM-managed dedicated IPs work, and when to use them.
tags: dedicated ip email deliverability tem ip management
dates:
  validation: 2024-03-10
  posted: 2024-03-10
categories:
  - transactional-email
---

<Macro id="dedicated-ip" />

The following dedicated IP management actions are included with the TEM Scale plan:
- **Automatic IP Warm-up** - Controlled progression of sending volumes to establish a good reputation.
    <Message type="tip">

>       Refer to the [IP warm-up information](/transactional-email/concepts#ip-warm-up) learn more about the warm-up process.

Reviewer: ```suggestion
      Refer to the [IP warm-up information](/transactional-email/concepts#ip-warm-up) to learn more about the warm-up process.
```

- **Optimization of sends** - Automatic adjustments to maximize reception rates.

<Message type="important">
  The Scaleway TEM team will intervene as much as possible to improve and maintain the domain's reputation score. However, since the TEM team does not own the domain, and has no control over traffic, they cannot commit to the complete resolution of incidents.
</Message>

## When to use a dedicated IP?

Dedicated IPs are useful for managing:

- **High volumes** - If you regularly send a large volume of emails (more than 1000 emails per day), a dedicated IP helps stabilize and improve deliverability.
- **Reputation** - Dedicated IPs help you avoid being affected by the performance of other senders on a shared IP.
- **Sensitive sends** - Cases such as financial transactions, healthcare, public, defense, or critical communications where confidentiality or reputation are crucial.
- **Compliance and security** - Regulatory compliance and maintaining the security of your email sending infrastructure, including obtaining necessary certifications and whitelisting your email sending IPs.

## When is a dedicated IP not necessary?

> A dedicated IP is not always advantageous, and in some cases, it can even harm deliverability:

Reviewer 2: ```suggestion
A dedicated IP is not always beneficial and, in some cases, may even negatively impact deliverability:
```
